app/views/application/_header.html.erb

```diff
@@ -1,18 +1,25 @@
 <header class="usa-header usa-header--basic">
-  <div class="usa-nav-container">
-    <div class="usa-navbar">
```

Comment:
the actual USWDS header component is very focused around navigation, so I moved towards using the grid and flexbox classes to make this look right. There may/probably are better ways to do this? I don't know!

Reply:
I think this is a fine way to do it as long as we remember to switch back to a navbar if/when we add some navigation to the header.